decidim-admin/app/views/decidim/admin/participatory_process_steps/index.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= render layout: 'decidim/admin/participatory_processes/layout' do %>
+  <%= render partial: 'table' %>
```

Comment:
Why not directly rendering the table here? I see so value in splitting this into a partial.